Fix scroll behavior of top level tabs in mobile navigation

bierik opened the pull request:
⚠️ Depends on https://github.com/4teamwork/ftw.theming/pull/76

- Introduce arrow on both sides of the tab navigation to show the user
  in which direction is more content
- Scroll to the active element when the mobile navigation is opened
- Keep scroll position when selecting tabs

![mobile_navi](https://cloud.githubusercontent.com/assets/1637820/25956471/1f6f671a-366c-11e7-83aa-50f9a61ec750.gif)

Commit:
Fix scroll behavior of top level tabs in mobile navigation.

- Introduce arrow on both sides of the tab navigation to show the user
  in which direction is more content

- Scroll to the active element when the mobile navigation is opened

- Keep scroll position when selecting tabs

Commit:
Fix linting errors

maethu reviewed (approved):
This looks great! If a customer wants the arrows clickable we gonna investigate this later :-)

👍

jone reviewed (approved):
👍 I like it

Commit:
Fix scroll bug on iOS devices in mobile menu.

Because iOS does not support `touch-action: pan-y`, the horizontal scroll
of the top level tabs was not working properly.
So attach the hammerjs swipe listener to the overlay element instead of the whole body.
